pkg/sqlite/load.go

```diff
@@ -192,86 +199,103 @@ func (s *SQLLoader) AddPackageChannels(manifest registry.PackageManifest) error
 			// Get CSV for current entry
 			channelEntryCSV, err := s.getCSV(tx, channelEntryCSVName)
 			if err != nil {
-				return err
+				errs = append(errs, err)
+				break
 			}
 
 			if err := s.addProvidedAPIs(tx, channelEntryCSV, currentID); err != nil {
-				return err
+				errs = append(errs, err)
+				break
```

Comment:
We should be able to continue processing even if the provided apis can't be added. It just means the registry won't be able to answer dependency questions about this particular bundle.